source/_integrations/bsblan.markdown

```diff
@@ -12,31 +13,32 @@ ha_domain: bsblan
 ha_platforms:
   - climate
   - diagnostics
+  - sensor
 ha_integration_type: device
 ---
 
 The **BSB-Lan** {% term integration %} integrates [BSBLan](https://github.com/fredlcore/BSB-LAN) devices into Home Assistant.
 
-BSBLan is a device that is made by `Frederik Holst` and `Ulf Dieckmann` for documentation and with
+BSBLan is a device that is made by `Frederik Holst` and with
 the help of many other contributors.
-The new board v3 is designed for an Arduino Due with an Ethernet-Shield for web-based controlling
+The board v3 is designed for an Arduino Due with an Ethernet-Shield for web-based controlling
 of heating systems such as `Elco Thision`, `Brötje` and similar systems.
+Also, available is an ESP32 version of the board.
 
 It can interface with the heating system over Boiler-System-Bus, Local Process Bus and PPS (Punkt-zu-Punkt Schnittstelle)
-For more information of which system it supports, have a look at their [documentation](https://1coderookie.github.io/BSB-LPB-LAN_EN/).
+For more information of which system it supports, take a look at their [documentation](https://docs.bsb-lan.de).
 
 {% include integrations/config_flow.md %}
 
+{% include integrations/config_flow.md %},
 For authentication HTTP authentication using a username and password,
 or using a passkey is supported. Use either one.
 
-For more documentation of the BSBLan device, check the [manual](https://1coderookie.github.io/BSB-LPB-LAN_EN/).
+For more documentation of the BSBLan device, check the [manual](https://docs.bsb-lan.de).
 
-To see a more detailed listing of the reported systems which are successfully used with BSB-LAN please follow the corresponding link:
+To see a more detailed listing of the reported systems which are successfully used with BSB-LAN, please follow the corresponding link:
 
-- [`Brötje`](https://1coderookie.github.io/BSB-LPB-LAN_EN/chap03.html#311-broetje)
-- [`Elco`](https://1coderookie.github.io/BSB-LPB-LAN_EN/chap03.html#312-elco)
-- [`Other Manufacturers (e.g. Fujitsu, Atlantic, Weishaupt)`](https://1coderookie.github.io/BSB-LPB-LAN_EN/chap03.html#313-other-manufacturers)
+[Supported heating systems](https://docs.bsb-lan.de/supported_heating_systems.html)
 
-The integration is tested with the stable firmware version `1.00`. A newer firmware versions will not work, because the parameters are changed of the specific info that is needed.
-Please use the latest release. [release 1.0](https://github.com/fredlcore/BSB-LAN/releases/tag/v1.0)
+The integration is tested with the stable firmware version `3.1.6-20230327101530`. A newer firmware versions may not work because the api could have changed.
```

Comment:
```suggestion
The integration is tested with the stable firmware version `3.1.6-20230327101530`. A newer firmware version may not work because the API could have changed.
```